File: app/src/main/java/arraymethod/ex02.java
package arraymethod;

public class ex02 {
    static int sum(int... numbers) {
        int result = 0;
        for (int n : numbers) {
            result += n;
        }
        return result;
    }

    public static void main(String[] args) {
        System.out.println("sum(1, 2, 3, 4, 5) = " + sum(1, 2, 3, 4, 5));
    }
}

File: app-kotlin/src/main/kotlin/arraymethod/ex02.kt
package arraymethod

class ex02 {
    companion object {
        fun sum(vararg numbers: Int): Int = numbers.sum()

        @JvmStatic
        fun main(args: Array<String>) {
            println("sum(1, 2, 3, 4, 5) = ${sum(1, 2, 3, 4, 5)}")
        }
    }
}
